cluster/juju/layers/kubernetes/templates/master.json

```diff
@@ -38,7 +38,7 @@
               "--etcd-certfile={{ etcd_cert }}",
               {%- endif %}
               "--etcd-servers={{ connection_string }}",
-              "--admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,DefaultStorageClass,ResourceQuota",
```

Comment:
I'm still perplexed at how this suddenly came up missing in all the hyperkube images.